src/app/projects/services/projects.service.ts

```diff
@@ -152,27 +168,27 @@ export class ProjectsService implements OnDestroy {
       return false;
     }
     switch (filters.location) {
-      case 'HCA Data Portal':
+      case ProjectsService.allowedLocations[0]:
```

Comment:
I wouuld suggest to use an enum or a dictionary with named keys to increase readability.